Source/Widgets/Cesium3DTilesInspector/Cesium3DTilesInspectorViewModel.js

```diff
@@ -711,6 +737,15 @@ define([
     });
 
     /**
+     * Returns the url to the feature
+     * @param {Cesium3DTileFeature} feature The feature
+     * @returns {String} The url to that feature's tile
+     */
+    Cesium3DTilesInspectorViewModel.prototype.getFeatureUrl = function(feature) {
```

Comment:
Sorry if there was miscommunication, I would much rather see `feature.content.url`, Since `Cesium3DTileContent` takes `url` as one of it's constructor parameters, we should definitely expose it, same for `Cesium3DTileFeature`.  In general, a class should always expose the core components that it takes in it's primary constructor as properties.